BWEnv/src/controller.cc

```diff
@@ -578,6 +578,7 @@ int Controller::getAttackFrames(int unitID) {
   int unitType = BWAPI::Broodwar->getUnit(unitID)->getType().getID();
   // From
   // https://docs.google.com/spreadsheets/d/1bsvPvFil-kpvEUfSG74U3E5PLSTC02JxSkiR8QdLMuw/edit#gid=0
+  // Photon Cannons may also have a value.
```

Comment:
Hm?

Reply:
From personal experience. The relevance of this delay is that you can cancel the attack of a unit which has already started attacking by issuing any commands. I've encountered the same problem with Photon Cannons, which aren't listed in that spreadsheet.

Reply:
I see, I was mostly confused by the comment. Maybe you can add a sentence with some context?

Reply:
About 15 minutes ahead of you :) 
![image](https://user-images.githubusercontent.com/116363/33332794-37a64d8e-d433-11e7-87e1-230b584937f0.png)

Reply:
Also adding:
// This value is primarily relevant for knowing how many frames a unit needs to be allowed to
// peform its attack animation before it can receive another command.
// Receiving a command before that period will cancel the attack.